sources/Licenses/Licenses.yml

```diff
@@ -0,0 +1,53 @@
+---
+extends: substitution
+ignorecase: true
+level: warning
+message: Use '%s' instead of '%s'
+action:
+  name: replace
+swap:
+  # Apache[ -]?2(?:\.0)?: Apache 2.0 # See notes, will be added later.
+  "[0O]BSD": 0BSD
+  BSD: BSD
+  BSD[ -]?1: BSD-1
+  BSD[ -]?2: BSD-2
+  BSD[ -]?3: BSD-3
```

Comment:
I found a lot of BSD-3-Clause by looking at source

I think we should at least add the following to the good/bad

BSD-3-Clause
BSD-3 Clause
BSD 3 Clause


https://github.com/search?q=language%3Amarkdown+%22bsd+3+clause%22&type=code

https://github.com/search?q=language%3Amarkdown+%22bsd-3+clause%22&type=code

https://github.com/search?q=language%3Amarkdown+%22bsd-3-clause%22&type=code

Reply:
> BSD-3-Clause 
>
>  BSD-3 Clause

I can add these two to good.txt since they should both be allowed under the rules.

> BSD 3 Clause

This one's wrong, so I can add this to bad.text